src/app/containers/PageHandlers/withData.test.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { shouldShallowMatchSnapshot } from '../../../testHelpers';
-import { articleDataNews } from '../Article/fixtureData';
+import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
 import WithData from './withData';
+import fronPageDataPidgin from '../../../../data/prod/pidgin/frontpage';
```

Comment:
Missing 't' in `fronPageDataPidgin`. Good idea to correct variable typos to avoid confusion in the event we search for `frontPageDataPidgin` and don't get results.